packages/app/src/components/presets/apple.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-use-before-define */
+
+import Inline from '../inline.macro'
+import { Box, Image, Text } from './scope'
+
+const code = (
+  <Inline>
+    <>
+      <Box
+        sx={{
+          position: 'relative',
+          background: query.bg,
+          fontFamily: '-apple-system, BlinkMacSystemFont, Arial, sans-serif',
+          px: 78,
+          py: 74
+        }}
+      >
+        <Image
+          sx={{ maxHeight: 36, display: 'block', mb: 14, ml: '2px' }}
+          src={query.logo}
+        />
+
+        <Text
+          sx={{
+            display: 'block',
+            fontSize: 46,
+            fontWeight: 200,
+            lineHeight: 1.05,
+            letterSpacing: 0.5,
+            color: query.color,
+            mb: 56
+          }}
+          dangerouslySetInnerHTML={{ __html: query.title }}
```

Comment:
What do you think about this @Kikobeats?

Reply:
It's okay, we do that in other templates as well

https://github.com/microlinkhq/cards/blob/fe8ee2417da51fd060e53df55cc4c31cb607dca7/packages/app/src/components/presets/kfund.js#L34